app/javascript/common/messages.js

```diff
@@ -9,11 +9,27 @@ export default {
       emptyPlanCardInfo: 'Create a migration plan to start migrating.',
       emptyPlanCardButton: 'Create Migration Plan'
     },
-    mappingWizard: {
-      title: 'Infrastructure Mapping Wizard',
+    wizard: {
       cancel: 'Cancel',
       back: 'Back',
-      next: 'Next'
+      next: 'Next',
+      close: 'Close'
+    },
+    mappingWizard: {
+      title: 'Infrastructure Mapping Wizard',
```

Comment:
why not to extract the strings using ```__('Infrastructure Mapping Wizard)```?

@priley86 did I miss understand how we handling i18n? (sorry was away for a few days).

Reply:
hey @ohadlevy would you like to use i18n.gettext the same way which Foreman does here for text?
https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/i18n.js

I somehow missed this was being done already when looking at the other i18n PR for Dates. I can quickly switch to this format if it works better here in mIQ (although I think that using `react-intl` messages is a better long term solution than exposing it as a global on the window).  Can you detail your plans a bit further there?

Happy to adjust this but I think it's outside the scope of this PR and we can fix it in a subsequent one.

Reply:
The `__("Text")` usage involves 2 things -

- extraction
A rake task that extracts all the strings with `__("...")` from the repo, which are then sent to the translators

- translation
The actual translation itself. All instances of `__("...")` would be substituted with the actual text from the language locale files that we receive from the translators

@priley86 With `react-intl`, do we need to explicitly extract the strings out?
With the `__("Text")` approach, we can skip the extraction step since we already have a process in place to do that.

Reply:
hey @AparnaKarve - thanks for clarifying. I was slightly confused on this.

If it's all done via Rake tasks now and that process works, it's OK with me. But maybe you can weigh how much additional effort it would be to add `babel-plugin-react-intl` to do the extraction like the guide [here](https://medium.freecodecamp.org/internationalization-in-react-7264738274a0) does? (yes, `react-intl` would also do the extraction, but additional effort).

I'm really torn on using both i18n approaches (`react-intl` for dates and numbers, and using `gettext` for messages). But will trust your best judgement here...

Reply:
@priley86 I think the process of extracting strings with `babel-plugin-react-intl`  might be comparable with the extraction process in Service UI where we use the `translate` angular directive.
```js
<span class="heading-class">{{notificationGroup.heading | translate}}</span>
```

@AllenBW and @chalettu may have some insight into that.

Reply:
keeping @mzazrivec in the loop too.